IPython/html/static/notebook/js/kernelselector.js

```diff
@@ -78,7 +78,7 @@ define([
             new_notebook_submenu.append(
                 $("<li>").attr("id", "new-notebook-submenu-"+ks.name).append(
                     $('<a>')
-                        .attr('href', '#')
+                        .attr('href', 'javascript:void(0)')
```

Comment:
That's fair. These page layout changes are complicated enough that it helps to avoid extra changes.